Frameworks/QuartzCore/CALayer.mm

```diff
@@ -353,19 +355,21 @@ void ReleaseDisplayTexture(DisplayTexture* tex) {
 static LockingBufferInterface _globallockingBufferInterface;
 
 CGContextRef CreateLayerContentsBitmapContext32(int width, int height, float scale) {
-    DisplayTexture* tex = NULL;
-
     if ([NSThread isMainThread]) {
-        tex = GetCACompositor()->CreateWritableBitmapTexture32(width, height);
-    }
+        DisplayTexture* tex = GetCACompositor()->CreateWritableBitmapTexture32(width, height);
+        if (!tex) {
+            return nil;
+        }
 
-    CGContextRef ret = _CGBitmapContextCreateWithTexture(width, height, scale, tex, &_globallockingBufferInterface);
+        IDisplayTextureImpl* displayTexture = new IDisplayTextureImpl(tex);
```

Comment:
Given it's being sent across dll boundaries, rather not

---
In reply to: [87050706](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87050706) [](ancestors = 87050706)